erpnext/stock/report/batch_wise_balance_history/batch_wise_balance_history.py

```diff
@@ -45,29 +45,30 @@ def get_columns(filters):
 
 
 def get_conditions(filters):
-	conditions = ""
+	conditions = []
 	if not filters.get("from_date"):
 		frappe.throw(_("'From Date' is required"))
 
 	if filters.get("to_date"):
-		conditions += " and posting_date <= '%s'" % filters["to_date"]
+		conditions.append("and posting_date <= %(to_date)s")
```

Comment:
```suggestion
		conditions.append(" and posting_date <= %(to_date)s")
```

Reply:
@sahil28297 this space not needed. oterwise two space one from here and anoter from join will get added

Reply:
joiin doesn't add a space, but there's already a space added in the sql statement before condition so yeah, we don't need it.